recipes/xtensor/run_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
If that is a place holder for the feedstock it is fine to leave it here. But I am not sure the env creation will work at test time.